validator/testcases/javascript/predefinedentities.py

```diff
@@ -11,52 +12,88 @@
+        "is forbidden under most circumstances."),
+    "editors_only": True,
+    "signing_severity": "high",
+}
 
 BANNED_PREF_BRANCHES = (
+    (u"app.update.", SECURITY_PREF_MESSAGE),
     (u"browser.newtab.url", CUSTOMIZATION_PREF_MESSAGE),
     (u"browser.newtabpage.enabled", CUSTOMIZATION_PREF_MESSAGE),
     (u"browser.preferences.instantApply", None),
-    (u"browser.search.defaultenginename", CUSTOMIZATION_PREF_MESSAGE),
-    (u"browser.search.searchEnginesURL", CUSTOMIZATION_PREF_MESSAGE),
+    (u"browser.search.defaultenginename", SEARCH_PREF_MESSAGE),
+    (u"browser.search.searchEnginesURL", SEARCH_PREF_MESSAGE),
     (u"browser.startup.homepage", CUSTOMIZATION_PREF_MESSAGE),
     (u"capability.policy.", None),
     (u"extensions.alwaysUnpack", None),
-    (u"extensions.blocklist.", None),
+    (u"extensions.blocklist.", SECURITY_PREF_MESSAGE),
     (u"extensions.bootstrappedAddons", None),
     (u"extensions.checkCompatibility", None),
     (u"extensions.dss.", None),
-    (u"extensions.getAddons.", None),
+    (u"extensions.getAddons.", SECURITY_PREF_MESSAGE),
     (u"extensions.getMoreThemesURL", None),
     (u"extensions.installCache", None),
     (u"extensions.lastAppVersion", None),
     (u"extensions.pendingOperations", None),
-    (u"extensions.update.", None),
+    (u"extensions.update.", SECURITY_PREF_MESSAGE),
     (u"general.useragent.", None),
-    (u"keyword.URL", CUSTOMIZATION_PREF_MESSAGE),
-    (u"keyword.enabled", CUSTOMIZATION_PREF_MESSAGE),
-    (u"network.proxy.autoconfig_url",
-        "As many add-ons have reason to change the proxy autoconfig URL, and "
-        "only one at a time may do so without conflict, extensions must "
-        "make proxy changes using other mechanisms. Installing a proxy "
-        "filter is the recommended alternative: "
-        "https://developer.mozilla.org/en-US/docs/Mozilla/Tech/XPCOM/"
-        "Reference/Interface/nsIProtocolProxyService#registerFilter()"),
-    (u"network.http.", None),
-    (u"network.websocket.", None),
+    (u"keyword.URL", SEARCH_PREF_MESSAGE),
+    (u"keyword.enabled", SEARCH_PREF_MESSAGE),
+    (u"network.proxy.autoconfig_url", {
+        "description": (
+            "As many add-ons have reason to change the proxy autoconfig URL, "
+            "and only one at a time may do so without conflict, extensions "
+            "must make proxy changes using other mechanisms. Installing a "
+            "proxy filter is the recommended alternative: "
+            "https://developer.mozilla.org/en-US/docs/Mozilla/Tech/XPCOM/"
+            "Reference/Interface/nsIProtocolProxyService#registerFilter()"),
+        "signing_severity": "low"}),
+    (u"network.proxy.", NETWORK_PREF_MESSAGE),
+    (u"network.http.", NETWORK_PREF_MESSAGE),
+    (u"network.websocket.", NETWORK_PREF_MESSAGE),
     (u"nglayout.debug.disable_xul_cache", None),
+    # Let's see if we can get away with this...
```

Comment:
On Tue, May 05, 2015 at 08:09:14AM -0700, Mathieu Agopian wrote:
>>      (u"nglayout.debug.disable_xul_cache", None),
>> +    # Let's see if we can get away with this...
>
>What do you mean?

I'm slightly worried that this substring may come up in 
too many unrelated contexts, but I'd prefer not to make it more 
specific, since basically any preference in that branch should 
be flagged.